ProefProeveVeluwe/Assets/Scripts/Runtime/Controls/GyroController.cs

```diff
@@ -0,0 +1,41 @@
+using UnityEngine;
+
+public class GyroController : MonoBehaviour
+{
+    private bool _gyroEnabled;
+    private Gyroscope _gyroscope;
+
+    [SerializeField] private GameObject _cameraContainer;
+    private Quaternion _rotation;
```

Comment:
Keep private variables togheter and don't add publics in between for consistency